Bender.yml

```diff
@@ -14,6 +14,7 @@ dependencies:
   reqrsp_interface:     { path: "hardware/deps/reqrsp_interface"                                            }
   snitch:               { path: "hardware/deps/snitch"                                                      }
   tech_cells_generic:   { git: "https://github.com/pulp-platform/tech_cells_generic.git",   version: 0.2.5  }
+  fpnew:                { git: "https://github.com/pulp-platform/cvfpu.git",                rev: pulp-v0.1.3 }
```

Comment:
I would rather have it here because it is also in the Tile as a standalone division and square-root unit. Plus, it is part of the mempool_cc, integrations in other Snitch core-complexes might differ.

Reply:
Ok that's true.